release-notes.txt

```diff
@@ -1,3 +1,510 @@
+* boards/nucleo-l053r8: MCU table addition to doc page (#20804)
+* boards/stm32l0538-disco: addition of pinout to documentation page (#20810)
+* build system: document `riotbuild.h` and deprecated `RIOT_MCU` (#20566)
+* doc/doxygen: print link to generated documentation (#20896)
+* doc: Point out common problem installing c2rust (#20844)
+* sys/net/app/cord: update doc (#20921)
+
+Build System / Tooling
+----------------------
+
++ Makefile: add .DEFAULT target to further guide new users (#20895)
+* dist/tools/bootterm: bump to 0.5 (#20871)
+* dist/tools/cosy: take BUILD_DIR into account (#20789)
+* make: export DEVELHELP (#20889)
+* makefiles/arch/native: don't be pedantic (#20873)
+* makefiles/suit: make use of `SUIT_SEC_PASSWORD` optional (#20862)
+* makefiles/suit: store public keys, make it easier to work with
+  multiple keys (#20858)
+
+
+Examples
+--------
+
++ examples/leds_shell: add example for interactive LEDs/GPIO control (#20782)
+* examples/gcoap-rust: Expose more functionality (#20838)
+* examples/leds_shell: use periph_gpio_mock on native (#20796)
+
+Testing
+-------
+
++ tests/pkg/emlearn: add model.h to repo (#20841)
+* gh-actions: bump deprecated packages (#20892)
+* tests/gnrc_ipv6_nib: rtr_ltime test (#20372)
+
+API Changes
+-----------
+
+* cpu/stm32: implement `periph_gpio_ll_switch_dir` (#20805)
+* drivers/periph_gpio: let gpio_read() return bool (#20936)
+* drivers/periph_gpio_ll: change API to access GPIO ports (#20639)
+
+Uncategorized
+-------------
++ .clang-format: Add default clang-format configuration (#20865)
++ CODING_CONVENTIONS.md: Add preprocessor directive formatting (#20866)
+* riot.doxyfile: Update and remove deprecated options (#20897)
+
+And 26 minor changes.
+
+Deprecations
+============
+
+
+Removals (1)
+------------
+
+- sys/ztimer: Remove the deprecated ztimer_now64 (#20826)
+
+Bug fixes (27)
+==============
+
+* boards/common/arduino-zero: Add Arudino SPI mapping (#20884)
+* cpu/native/periph_timer: add missing -lrt to linker flags (#20816)
+* cpu/sam0_common/periph_adc: add work around for errata 2.1.6
+  [backport 2024.10] (#21001)
+* bluetil: Ensure advertisement length does not exceed pkt len (#20881)
+* boards/samd5x: GCLK source for ADC must not exceed 100 MHz (#20932)
+* boards/stm32f429i-disc1: fix touch screen axis [backport 2024.10] (#21014)
+* build system: simplify docker image pinning (#20877)
+* core/mutex: use thread_yield_higher() in mutex_unlock() (#20890)
+* core/panic: don't use LOG_ functions in panic handler (#20790)
+* cpu/sam0_common/periph_gpio_ll: fix gpio_get_port() and
+  gpio_ll_query_conf() [backport 2024.10] (#21008)
+* dns_msg: skip RDLENGTH_LENGTH field when skipping record (#20857)
+* drivers/at86rf215: return ENETDOWN when interface is down [backport
+  2024.10] (#21037)
+* examples/lorawan: drop crazy STM32 hack (#20883)
+* gcoap/forward_proxy: handle timeout case (#20915)
+* gh-actions: bump upload-artifact to v4 (#20875)
```

Comment:
Why is this here but `gh-actions: bump deprecated packages (#20892)` is under `Testing`?